src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -310,13 +351,15 @@ g.test('copy_rectangle')
         break;
       }
       case 2: {
-        textureSize.depthOrArrayLayers = textureSizeValue;
+        textureSize.depthOrArrayLayers =
+          dimension === '2d' ? textureSizeValue : textureSizeValue << mipLevel;
```

Comment:
nit: flip this so it'll still be correct when 1d is added:
```suggestion
          dimension === '3d' ? textureSizeValue << mipLevel : textureSizeValue;
```

Reply:
Done.